internal/core/command.go

```diff
@@ -82,6 +82,10 @@ func (c *Command) Override(builder func(command *Command) *Command) {
 }
 
 func (c *Command) getPath() string {
+	if c == nil {
```

Comment:
Maybe we should test the command before calling `getPath` ([here](https://github.com/scaleway/scaleway-cli/pull/660/files#diff-110b5802494c0ef700f188b5202069b1R65)) and remove this condition.